psi4/src/psi4/libfock/cubature.h

```diff
@@ -91,14 +92,16 @@ class MolecularGrid {
     /// Orientation matrix
     std::shared_ptr<Matrix> orientation_;
     /// Radial grids, per atom
-    std::vector<std::shared_ptr<RadialGrid> > radial_grids_;
+    std::vector<std::shared_ptr<RadialGrid>> radial_grids_;
     /// Spherical grids, per atom and radial point
-    std::vector<std::vector<std::shared_ptr<SphericalGrid> > > spherical_grids_;
-    /// index_[fast_index] = slow_index
-    int* index_;
+    std::vector<std::vector<std::shared_ptr<SphericalGrid>>> spherical_grids_;
+    /// Grid points, per atom
```

Comment:
Please add a comment somewhere to say that every `MolecularGrid` should have `atomic_grids_`, but whether it has `atomic_blocks_` is up to what blocking scheme(s) the user asked for.

Reply:
I tried to clarify the situation with atomic_grids/blocks in their respective comments.